linux-anvil-alma/Dockerfile

```diff
@@ -0,0 +1,93 @@
+ARG DISTRO_ARCH
+ARG DISTRO_NAME
+ARG DISTRO_VER
+FROM --platform=linux/${DISTRO_ARCH} ${DISTRO_NAME}:${DISTRO_VER}
+
+LABEL maintainer="conda-forge <conda-forge@googlegroups.com>"
+
+ARG DISTRO_ARCH
+ARG DISTRO_NAME
+ARG DISTRO_VER
+ENV DISTRO_ARCH=${DISTRO_ARCH} \
+    DISTRO_NAME=${DISTRO_NAME} \
+    DISTRO_VER=${DISTRO_VER}
+
+# Set an encoding to make things work smoothly.
+ENV LANG en_US.UTF-8
+
+# Add qemu in here so that we can use this image on regular linux hosts with qemu user installed
+ADD qemu-aarch64-static /usr/bin/qemu-aarch64-static
+ADD qemu-arm-static /usr/bin/qemu-arm-static
+ADD qemu-ppc64le-static /usr/bin/qemu-ppc64le-static
+ADD qemu-s390x-static /usr/bin/qemu-s390x-static
+
+# bust the docker cache so that we always rerun the installs below
+ADD https://loripsum.net/api /opt/docker/etc/gibberish
+
+# Resolves a nasty NOKEY warning that appears when using yum.
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
+
+# Install basic requirements.
+COPY scripts/yum_clean_all /opt/docker/bin/
+RUN yum update -y && \
+    yum install -y \
+        bzip2 \
+        glibc-minimal-langpack \
+        glibc-langpack-en \
+        sudo \
+        tar \
+        which && \
+    /opt/docker/bin/yum_clean_all
+
+# Run common commands
+COPY scripts/run_commands /opt/docker/bin/run_commands
+RUN /opt/docker/bin/run_commands
+
+# Download and cache new compiler packages.
+# Should speedup installation of them on CIs.
+RUN source /opt/conda/etc/profile.d/conda.sh && \
+    conda activate && \
+    conda create -n test --yes --quiet --download-only \
+        conda-forge::binutils_impl_linux-64 \
+        conda-forge::binutils_linux-64 \
+        conda-forge::gcc_impl_linux-64 \
+        conda-forge::gcc_linux-64 \
+        conda-forge::gfortran_impl_linux-64 \
+        conda-forge::gfortran_linux-64 \
+        conda-forge::gxx_impl_linux-64 \
+        conda-forge::gxx_linux-64 \
+        conda-forge::binutils_impl_linux-aarch64 \
+        conda-forge::binutils_linux-aarch64 \
+        conda-forge::gcc_impl_linux-aarch64 \
+        conda-forge::gcc_linux-aarch64 \
+        conda-forge::gfortran_impl_linux-aarch64 \
+        conda-forge::gfortran_linux-aarch64 \
+        conda-forge::gxx_impl_linux-aarch64 \
+        conda-forge::gxx_linux-aarch64 \
+        conda-forge::binutils_impl_linux-ppc64le \
+        conda-forge::binutils_linux-ppc64le \
+        conda-forge::gcc_impl_linux-ppc64le \
+        conda-forge::gcc_linux-ppc64le \
+        conda-forge::gfortran_impl_linux-ppc64le \
+        conda-forge::gfortran_linux-ppc64le \
+        conda-forge::gxx_impl_linux-ppc64le \
+        conda-forge::gxx_linux-ppc64le \
+        conda-forge::libgcc-ng \
+        conda-forge::libgfortran-ng \
+        conda-forge::libstdcxx-ng && \
```

Comment:
Why are you downloading all? This is going to be a very big image

Reply:
It was easier to lump them into one step. Plus in the cross-compilation case many of these get downloaded anyways

Though if someone wants to propose conditional logic, that would be reasonable

Reply:
Let's remove these lines. This cache does not work most of the time anyways. This downloads gcc 13 whereas the recipes uses gcc 12. For the past couple of years, it's very rare that this caching worked because we always had a higher gcc version available, but recipes used lower ones.